plugin/src/server/sc/plugin2021/GamePlugin.kt

```diff
@@ -0,0 +1,34 @@
+package sc.plugin2021
+
+import sc.api.plugins.IGameInstance
+import sc.api.plugins.IGamePlugin
+import sc.api.plugins.host.IGamePluginHost
+import sc.plugin2020.GamePlugin
+import sc.plugins.PluginDescriptor
+import sc.shared.ScoreDefinition
+
+@PluginDescriptor(name = "Hive", uuid = GamePlugin.PLUGIN_UUID, author = GamePlugin.PLUGIN_AUTHOR)
```

Comment:
Careful with the game id!

Reply:
Ah yeah, I forgot...
On that note, I probably forgot to change some `plugin2020` to `plugin2021` as well.
Thanks for the note!